content/docs/latest/spire-helm-charts-hardened-about/installation.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Installation
+short: Installation
+description: How to install the SPIRE stack using the SPIRE Helm charts
+kind: spire-helm-charts-hardened-about
+weight: 100
+aliases:
+    - /docs/latest/helm-charts-hardened/installation
+---
+
+## Non Production Deployment
+
+To do a quick non production install suitable for quick testing in something like minikube:
+
+```
+helm upgrade --install --create-namespace -n spire-server spire-crds spire-crds \
+ --repo https://spiffe.github.io/helm-charts-hardened/
+
```

Comment:
These are very helpful, but maybe we should include a few more parameters like 
```
--set global.spire.namespaces.create=true 
``` 
so the namespaces are correctly created based on our charts

Reply:
namespaces.create won't really work properly without enabling the recommendations and then disabling the ones that don't apply to a non production deployment.

The example here is to make it as easy as possible to get a workable deployment with as little config/steps as possible to let new users kick the tires. If there are some additional options that fit that use case, we should probably try and make those enabled by default.